leaguepedia_parser/parsers/game_parser.py

```diff
@@ -70,7 +83,7 @@ def get_tournaments(
     return [transmute_tournament(tournament) for tournament in result]
 
 
-def get_games(tournament_overview_page=None, **kwargs) -> List[LolGame]:
+def get_games(tournament_name=None, **kwargs) -> List[LolGame]:
```

Comment:
It is actually using the overview page field, sorry for not updating the doc. This should be reverted.